lib/browser/guest-view-manager.js

```diff
@@ -184,6 +184,32 @@ const attachGuest = function (embedder, elementInstanceId, guestInstanceId, para
     disableBlinkFeatures: params.disableblinkfeatures
   }
 
+  // parse the 'webpreferences' attribute string, if set
+  // this uses the same parsing rules as window.open uses for its features
```

Comment:
I factored out the function into the `/lib/common` dir, but I can't seem to make the `require('../common/parse-features-string')` happen in `/lib/browser/guest-view-manager.js`. I'm guessing this is due to changes to the `require` search paths. Any suggestions?